Guevados/jamMetroidvania
# Feature/pj attack animation

# Problem

As PJ I want to be animated on attacks

# Solution 

Handle animations for PJ when attacks

# How to test

Run game and check left right movement animation pressing keys LEFT / RIGHT / D

https://github.com/Guevados/jamMetroidvania/assets/28446213/4d77dc74-628e-4ed4-b637-52a09c52f42d
